src/cortxfs/cortxfs_fops.c

```diff
@@ -115,6 +121,8 @@ int cfs_creat_ex(struct cfs_fs *cfs_fs, cfs_cred_t *cred, cfs_ino_t *parent,
 		(void) cfs_unlink(cfs_fs, cred, parent, &object, name);
 		(void) kvs_discard_transaction(kvstor, &index);
 	}
+	perfc_trace_finii(PEA_CFS_RES_RC);
```

Comment:
What does it mean? Are you trying to use perfc_trace_finii as perfc_trace_attr?

Reply:
Fixed.

-bash-4.2$ ls -lrt / | grep addb
drwx------    3 root root     4096 Sep 29 00:50 addb_25843
drwx------    3 root root     4096 Sep 29 00:50 addb_25920
drwx------    3 root root     4096 Oct 19 01:59 addb_1010
drwx------    3 root root     4096 Oct 19 01:59 addb_1088
drwx------    3 root root     4096 Oct 19 02:19 addb_15253
drwx------    3 root root     4096 Oct 19 02:19 addb_15328
drwx------    3 root root     4096 Oct 29 09:14 addb_25868
drwx------    3 root root     4096 Oct 29 09:14 addb_25943
drwx------    3 root root     4096 Oct 29 22:14 addb_22928
drwx------    3 root root     4096 Oct 29 22:14 addb_23001
drwx------    3 root root     4096 Nov  3 00:09 addb_24435
drwx------    3 root root     4096 Nov  3 00:09 addb_24510
drwx------    3 root root     4096 Nov  3 00:23 addb_7900
drwx------    3 root root     4096 Nov  3 00:23 addb_7973
-bash-4.2$ sudo m0addb2dump -p /usr/lib64/libcortx-utils.so -- /addb_7973/o/100000000000000\:2 | grep fsuser > addb_dump_op_decode_6
-bash-4.2$ cat addb_dump_op_decode_6 | grep cfs_result_rc
* 2020-11-03-00:25:02.239037385 fsuser_attribute cfs_create, attribute,  64b,        cfs_result_rc, 0
* 2020-11-03-00:25:02.245876292 fsuser_attribute cfs_create_ex, attribute, 64a,      cfs_result_rc, 0
* 2020-11-03-00:25:02.281896665 fsuser_attribute cfs_create, attribute,  780,        cfs_result_rc, 0
* 2020-11-03-00:25:02.287954425 fsuser_attribute cfs_create_ex, attribute, 77f,      cfs_result_rc, 0
* 2020-11-03-00:25:02.367043898 fsuser_attribute cfs_create, attribute,  9ce,        cfs_result_rc, 0
* 2020-11-03-00:25:02.374253897 fsuser_attribute cfs_create_ex, attribute, 9cd,      cfs_result_rc, 0
* 2020-11-03-00:25:02.410446855 fsuser_attribute cfs_create, attribute,  af5,        cfs_result_rc, 0
* 2020-11-03-00:25:02.417331338 fsuser_attribute cfs_create_ex, attribute, af4,      cfs_result_rc, 0